assets/scripts/segments/drag_and_drop.js

```diff
@@ -212,11 +71,6 @@ export function onBodyMouseDown (event) {
   }
 
   store.dispatch(clearMenus())
-
-  if (el.classList.contains('drag-handle')) {
```

Comment:
I noticed that because we set the `resizeDragState` in `beginDrag`, we don't show the resize handles until a dragging action starts instead of once we click on a drag handle. Does it matter?

Reply:
Hmm. It's a subtle difference but it might matter. There's no indication of what happens now when you click the handle. I'll have to think about whether we can put it back. If I do, it won't be in the global mousedown handler (it'd probably be in the handle component itself).